common/dynamicconfig/constants.go

```diff
@@ -81,6 +81,30 @@ type (
 	}
 )
 
+// ListAllProductionKeys returns all key used in production
+func ListAllProductionKeys() []Key {
+	result := make([]Key, 0, len(IntKeys)+len(BoolKeys)+len(FloatKeys)+len(StringKeys)+len(DurationKeys)+len(MapKeys))
+	for i := TestGetIntPropertyFilteredByTaskListInfoKey + 1; i < LastIntKey; i++ {
```

Comment:
What do you think about also exposing methods for each of these as well which hides this implementation detail of the Last key and similar? 

`ListIntConfigKeys()` etc?

Reply:
I don't find a use case for listing keys by type now. I can add that later if I find a use case